src/pages/lib-v1/push-notifications/setup-push-service/q/platform/[platform].mdx

```diff
@@ -0,0 +1,27 @@
+export const meta = {
+  title: `Setting up push notification services`,
+  description: `Learn how to setup the various push notification services for your mobile app.`,
+  filterKey: 'platform',
+  supportedPlatforms: INTEGRATION_FILTER_OPTIONS
+};
+
+import { generateStaticPaths } from '@/utils/generateStaticPaths.tsx';
+
+import { INTEGRATION_FILTER_OPTIONS } from '@/utils/filter-data.ts';
+
+export const getStaticPaths = () => {
+  return generateStaticPaths(meta.filterKey, meta.supportedPlatforms);
+};
+
+export const getStaticProps = (context) => {
+  return {
+    props: {
+      platform: context.params.platform,
+      filterKind: meta.filterKey
+    }
+  };
+};
+
+import crossPlatformSetupService from '/src/fragments/lib/push-notifications/common/setup_push_service/cross-platform-setup.mdx';
```

Comment:
Intentionally targeting the shared `/lib/` fragment as changes to docs here should apply universally across versions